doc/API.md

```diff
@@ -470,6 +482,65 @@ const listenMa = libp2p.transportManager.getAddrs()
 // [ <Multiaddr 047f00000106f9ba - /ip4/127.0.0.1/tcp/63930> ]
 ```
 
+### addressManager.getAnnounceAddrs
+
+Get the multiaddrs that were provided to announce to the network.
+
+`libp2p.addressManager.getAnnounceAddrs()`
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Array<Multiaddr>` | Provided announce multiaddrs |
+
+#### Example
+
+```js
+// ...
+const announceMa = libp2p.addressManager.getAnnounceAddrs()
+// [ <Multiaddr 047f00000106f9ba - /dns4/peer.io/...> ]
+```
+
+### addressManager.getNoAnnounceAddrs
+
+Get the multiaddrs that were provided to not announce to the network.
+
+`libp2p.addressManager.getNoAnnounceAddrs()`
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Array<Multiaddr>` | Provided noAnnounce multiaddrs |
+
+#### Example
+
+```js
+// ...
+const noAnnounceMa = libp2p.addressManager.getNoAnnounceAddrs()
+// [ <Multiaddr 047f00000106f9ba - /ip4/127.0.0.1/tcp/63930> ]
+```
+
+### transportManager.getAddrs
+
+Get the multiaddrs that libp2p transports are using to listen on.
+
+`libp2p.transportManager.getAddrs()`
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Array<Multiaddr>` | listening multiaddrs |
+
+#### Example
+
+```js
+// ...
+const listenMa = libp2p.transportManager.getAddrs()
+// [ <Multiaddr 047f00000106f9ba - /ip4/127.0.0.1/tcp/63930> ]
+
```

Comment:
This whole added block already exists, this is adding duplicate data.

Reply:
Removing this should also fix the formatting below as it isn't properly closing the code blocks